README.md

```diff
@@ -496,7 +498,9 @@ you would need to do is gradually rename your jsx files to js files and everythi
 ```yaml
 webpack:
     loaders:
-        babel: ~
+        babel:
+            presets:
+                - "es2015"
```

Comment:
Please provide some information about what this is, what the possible settings are and/or link to a webpage with more information about this.